integration-cli/docker_cli_inspect_test.go

```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/docker/docker/api/types"
 	"github.com/docker/docker/api/types/container"
 	"github.com/docker/docker/integration-cli/cli"
+	"github.com/docker/docker/testutil/specialimage"
```

Comment:
Not sure if this is the right moment to do so (perhaps it is, because it's a new package); we should move more of these test utils `internal` (`testutil` as a whole should be internal again, but that for sure should be done separate).

I know we already created some in `internal`, e.g. https://github.com/moby/moby/tree/master/internal/testutils

Reply:
Yeah good idea. internal would be a good place for it. Will move it, unless you'd prefer to combine it with the frozen images code?